go.mod

```diff
@@ -4,12 +4,9 @@ go 1.24.1
 
 toolchain go1.24.2
 
-require github.com/amikos-tech/chroma-go v0.2.2
-
-// Till https://github.com/amikos-tech/chroma-go/pull/221 is merged.
-replace github.com/amikos-tech/chroma-go => github.com/Anush008/chroma-go v0.2.3
```

Comment:
This replace is breaking the build when I tried to do a release.

https://github.com/qdrant/migration/blob/main/CONTRIBUTING.md#releasing-a-new-version